akri/README.md

```diff
@@ -0,0 +1,53 @@
+We recommend Akri to enable leaf device access on the cluster.
+
+## Prerequisites:
+Before performing the procedure in this topic, you must have installed and configured the following:
+
+* A Kubernetes cluster created with AKS-HCI with at least 1 master and 1 Linux worker nodes.
+* Install Helm V3 and add it to system PATH. 
+  Helm is the package manager for Kubernetes that runs on a local machine with `kubectl` access to the Kubernetes cluster. The installation process for Prometheus and the Certificate Manager leverage Helm charts available on the public Helm repo. Please review the steps on your own recommended way of [Installing Helm](https://helm.sh/docs/using_helm/#installing-helm).

```

Comment:
Is the line about Prometheus relevant?

Reply:
no ... most of this is copy/paste and then modified ... if you see anything other than Akri, it is most likely NOT intentional.  thanks.  will remove.